index.html

```diff
@@ -23,16 +20,18 @@
   <body>
     <main class="main-Container">
       <section class="all-books">
-        <h1>All Awesome Books</h1>
-        <div class="listShow"></div>
+        <h1 class = "main-title">All Awesome Books</h1>
+        <div class="listShow">
+
+        </div>
       </section>
-      <hr class="horizontal-line1" />
+      <hr class="horizontal-line" />
       <section class="add-books">
-        <h2 class="add-title">Add a New Book</h2>
+        <h4 class="add-title">Add a New Book</h4>
         <div class="input-book">
           <form action="#">
-            <input type="text" class="title" placeholder="Title" required />
-            <input type="text" class="author" placeholder="Author" required />
+            <input type="text" class="title" placeholder="Title" required /><br>
+            <input type="text" class="author" placeholder="Author" required /><br>
```

Comment:
[OPTIONAL]

- I would suggest that after clicking on Add button it's better to clear the input field so that the user can put another data there and it will give a better UI.